hw/ip/spi_host/doc/checklist.md

```diff
@@ -16,15 +16,15 @@ All checklist items refer to the content in the [Checklist.]({{< relref "/doc/pr
 
 Type          | Item                           | Resolution  | Note/Collaterals
 --------------|--------------------------------|-------------|------------------
-Documentation | [SPEC_COMPLETE][]              | Not Started | [SPI_HOST Design Spec]({{<relref "hw/ip/spi_host/doc" >}})
-Documentation | [CSR_DEFINED][]                | Not Started |
-RTL           | [CLKRST_CONNECTED][]           | Not Started |
-RTL           | [IP_TOP][]                     | Not Started |
-RTL           | [IP_INSTANTIABLE][]            | Not Started |
-RTL           | [PHYSICAL_MACROS_DEFINED_80][] | Not Started |
-RTL           | [FUNC_IMPLEMENTED][]           | Not Started |
-RTL           | [ASSERT_KNOWN_ADDED][]         | Not Started |
-Code Quality  | [LINT_SETUP][]                 | Not Started |
+Documentation | [SPEC_COMPLETE][]              | Mostly Done | Programming model still needs work, noted in spec [SPI_HOST Design Spec]({{<relref "hw/ip/spi_host/doc" >}})
+Documentation | [CSR_DEFINED][]                | Done        |
+RTL           | [CLKRST_CONNECTED][]           | Done        |
+RTL           | [IP_TOP][]                     | Done        |
+RTL           | [IP_INSTANTIABLE][]            | Done        |
+RTL           | [PHYSICAL_MACROS_DEFINED_80][] | N/A         | Currently uses register FIFO's, may change to SRAM based
```

Comment:
@msfschaffner Can you have a look at that? If we really want to go to a macro we should plan for it soon I guess. (But if I remember the discussions right the cut-off point for SRAM is rather late, so probably FFs it is here?)

Reply:
That will have a big floorplan impact at this point, and would certainly cause issues on the Nuvoton side.
What configurations are we talking about?

Reply:
There are two 32w x64deep async fifos.   In addition, one of these has a small sync fifo after the async.  IIRC and looking at conversations such as in https://github.com/lowRISC/opentitan/issues/5112 this is near the crossover right?

Reply:
Yes this exactly around the crossover point (SRAMs could potentially be slightly denser). 
So given the floorplan impact I would refrain from making these SRAM macros now.

Reply:
Can you remove the comment?

Reply:
Thanks Michael, sure will do.